fix samtools commands, need to use -o to specify location of output file
author: Kenneth Hoste (HPC-UGent)
--- norgal.py.orig	2019-04-29 17:40:44.478335000 +0200
+++ norgal.py	2019-04-29 17:41:27.856070526 +0200
@@ -246,7 +246,7 @@
 doit(cmd,"bwa mem")
 stat += 1
 
-cmd = "{0}/samtools sort ref/genomeDNA.bam ref/genomeDNA.sort".format(bpath)
+cmd = "{0}/samtools sort -o ref/genomeDNA.sort.bam ref/genomeDNA.bam".format(bpath)
 doit(cmd,"initial_sort")
 stat += 1
 cmd = "{0}/samtools faidx ref/genomeDNA.sort.bam".format(bpath)
@@ -369,7 +369,7 @@
 doit(cmd,"Mapping reads to potential mtdna")
 stat += 1
 
-cmd = "{0}/samtools sort ref/circular.bam ref/circular.sort".format(bpath)
+cmd = "{0}/samtools sort -o ref/circular.sort.bam ref/circular.bam".format(bpath)
 doit(cmd,"sorting final mtdna")
 stat += 1
 cmd = "{0}/samtools faidx ref/circular.sort.bam".format(bpath)
